service/src/test/java/bio/terra/pipelines/controller/PublicApiControllerTest.java

```diff
@@ -60,4 +72,31 @@ void testVersion() throws Exception {
         .andExpect(jsonPath("$.github").value(github))
         .andExpect(jsonPath("$.build").value(build));
   }
+
+  @Test
+  void testIndex() throws Exception {
+    this.mockMvc.perform(get("/")).andExpect(status().is3xxRedirection());
+  }
+
+  @Test
+  void testGetSwagger() throws Exception {
+    var swaggerPaths = Set.of("/index.html", "/swagger-ui.html");
+    for (var path : swaggerPaths) {
+      this.mockMvc
+          .perform(get(path))
+          .andExpect(status().isOk())
+          .andExpect(model().attributeExists("clientId"));
+    }
+  }
+
+  // this test doesn't work because I can't figure out how to use the secondaryTemplateResolver bean
```

Comment:
happy to just take this out unless anyone knows how to make it work

Reply:
what endpoint do you expect to use the secondaryTempalteResolver?